docs/General/install.rst

```diff
@@ -39,7 +40,8 @@ A C and C++ compiler
     use a "K&R C" compiler. The C++ compiler must support C++ 17.
 
     Development of Freeciv21 is primarily done with :file:`gcc`, the GNU project's excellent C and C++
-    compiler. Microsoft Windows MS Visual C support is under development.
+    compiler. Microsoft Windows is supported with :file:`clang` and :file:`clang++` in
+    :doc:`Visual Studio <../Contributing/visual-studio>`.

```

Comment:
Might be worth mentioning that clang works on all platforms, also Linux and Apple.